tools/StaticAnalysis/Exceptions/Az.Batch/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,13 @@
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
```

Comment:
You cannot include breaking changes in a none-major release. Please revert all the breaking changes in this PR.

Reply:
The breaking changes were approved in the [design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1073), and this is intended to be a major release (4.0.0).

Reply:
> The breaking changes were approved in the [design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1073), and this is intended to be a major release (4.0.0).

The design review is approved in the last sprint. You can include your breaking changes into the major release of the last sprint. But since the PR doesn't catch the release, the breaking changes can only be released in the next major release, about May, 2022.

You can split your PR and then submit non-breaking change part first